addOns/authhelper/src/main/resources/db/migration/V1__Create_tables.sql

```diff
@@ -0,0 +1,65 @@
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "AUTHENTICATIONMETHOD" VARCHAR(4096) NOT NULL,
+    "CONTEXT" VARCHAR(4096) NOT NULL,
+    "USER" VARCHAR(4096) NOT NULL,
+);
+
+
+CREATE CACHED TABLE IF NOT EXISTS "AUTHHELPER_DIAGNOSTIC_STEP" (
+    "CREATETIMESTAMP" TIMESTAMP NULL,
+    "ID" BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    "DIAGNOSTICID" BIGINT NOT NULL,
+    "NUMBER" BIGINT NOT NULL,
+    "URL" VARCHAR(4096) NOT NULL,
```

Comment:
We're using a much bigger size for HistoryReference URIs in core: https://github.com/zaproxy/zaproxy/blob/8a59ce5e7a8821cdf5e697fea1b3a847c4d57e07/zap/src/main/resources/org/zaproxy/zap/resources/zapdb.script#L37

```
URI VARCHAR(1048576)
```

Reply:
That's because we tamper with path/query, which we are not going to do here but can increase just in case?